### PR TITLE
DINT-903: add rankingType and trendingWindow to search response schema

### PR DESCRIPTION
**Pull request by aniham**

### CODE REVIEW

File: packages/storefront-events-collector/src/contexts/searchResults.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ const createContext = (searchUnitId: string, searchResults?: SearchResults): Sea
         data: {
             searchUnitId: searchResultsUnit.searchUnitId,
             searchRequestId: searchResultsUnit.searchRequestId,
+            rankingType: searchResultsUnit.rankingType,
+            trendingWindow: searchResultsUnit.trendingWindow,
```
Comment (grahamcrackers) [inline packages/storefront-events-collector/src/contexts/searchResults.ts:30]: i know you are just adding these two fields but you could spread this over the data attribute, and then map the searchResults that don't match the data type like so
```ts
        data: {
            ...searchResultsUnit
            productCount: searchResultsUnit.products.length,
            categoryCount: searchResultsUnit.categories.length,
            suggestionCount: searchResultsUnit.suggestions.length,
        },
```

File: packages/storefront-events-collector/src/schemas.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ const schemas = {
     SEARCH_INPUT_SCHEMA_URL: "iglu:com.adobe.magento.entity/search-input/jsonschema/1-0-11",
     SEARCH_RESULT_CATEGORY_SCHEMA_URL: "iglu:com.adobe.magento.entity/search-result-category/jsonschema/1-0-1",
     SEARCH_RESULT_PRODUCT_SCHEMA_URL: "iglu:com.adobe.magento.entity/search-result-product/jsonschema/1-0-2",
-    SEARCH_RESULTS_SCHEMA_URL: "iglu:com.adobe.magento.entity/search-results/jsonschema/1-0-9",
+    SEARCH_RESULTS_SCHEMA_URL: "iglu:com.adobe.magento.entity/search-results/jsonschema/1-0-11",
```
Comment (grahamcrackers) [inline packages/storefront-events-collector/src/schemas.ts:15]: is this published to prod in snowplow?

**Review by grahamcrackers — COMMENTED** [review]
Have a question before Approval

File: packages/storefront-events-collector/src/schemas.ts (hunk 2 above)
Comment (aniham) [reply packages/storefront-events-collector/src/schemas.ts:15]: hi @grahamcrackers! yes it is!! thank you so much for remembering and for checking! i merged it to prod as part of [DINT-902](https://jira.corp.adobe.com/browse/DINT-902) - the new schema version is viewable [here](https://console.snowplowanalytics.com/f0aa80a7-0012-46d3-ad4e-5fbe36c22504/data-structures?hidden=false&rows=10&search=&sort=lastUpdate&sortO=desc). attaching screenshot too for completeness

Screenshot 2023-04-25 at 11 31 50

**Review by grahamcrackers — APPROVED** [review]
🚀

**Pull request merged**